### PR TITLE
Model Name and Description Documentation

### PR DESCRIPTION
**Pull request by EmosewaMC**
Adds documentation for Property naming protocols.

### CODE REVIEW

File: docs/game-mechanics/properties/naming-and-describing-models.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    Client -> WorldServer: [<b>Game Message UpdatePropertyOrModelForFilterCheck</b>] 
+
+    ==Property or Model Name/Description Has Been Moderated (reply can be in any order)==
+    WorldServer -> Client: [<b>Game Message SetName</b>] Sent to the model in the world
+    WorldServer -> Client: [<b>Item Component Serialization</b>] Description updated with moderated description
+    @enduml
+
+Networked message definitions:
+
+* Game Message `UpdatePropertyOrModelForFilterCheck <https://lcdruniverse.org/lu_packets/lu_packets/world/gm/server/struct.UpdatePropertyOrModelForFilterCheck.html>`_
```
Comment (Xiphoseer) [inline docs/game-mechanics/properties/naming-and-describing-models.rst:22]:
```suggestion
* Game Message :gm:server:`UpdatePropertyOrModelForFilterCheck`
```

File: docs/game-mechanics/properties/naming-and-describing-models.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+* Game Message `SetName <https://lcdruniverse.org/lu_packets/lu_packets/world/gm/client/struct.SetName.html>`_
```
Comment (Xiphoseer) [inline docs/game-mechanics/properties/naming-and-describing-models.rst:24]:
```suggestion
* Game Message :gm:client:`SetName`
```

**Review by Xiphoseer — COMMENTED** [review]
I think these shorthands for linking to lu_packets should work.

File: docs/game-mechanics/properties/naming-and-describing-models.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Component serialization:
+
+* Item Component `Serialization <https://lcdruniverse.org/lu_packets/lu_packets/raknet/client/replica/item/struct.ItemConstruction.html>`_
```
Comment (Xiphoseer) [inline docs/game-mechanics/properties/naming-and-describing-models.rst:28]:
```suggestion
* Item Component :packet:`raknet/client/replica/item/struct.ItemConstruction`
```

**Comment by EmosewaMC** [comment]
Oh cool didn't know these shorthands existed

**Comment by EmosewaMC** [comment]
Should be all good now if there is no further feedback on ways to improve!

**Pull request merged**